app/navigation/routes.js

```diff
@@ -167,6 +177,14 @@ export const Routes = {
         navigationProps: {
             title: {id: 'mobile.routes.user_profile', defaultMessage: 'Profile'}
         }
+    },
+    Saml: {
+        key: 'Saml',
+        transition: RouteTransitions.Horizontal,
+        component: Saml,
+        navigationProps: {
+            title: {id: 'mobile.routes.saml', defaultMessage: 'Single SignOn'}
```

Comment:
Do you mean something like `OneLogin`, `OKTA` etc? if that's the case yes we do, but dunno if we want to use that here, maybe we should talk about it in the feature standup meeting